README.md

```diff
@@ -106,22 +106,52 @@ the database constants are correct.
 
 ### wp config get
 
-Gets variables, constants, and file includes defined in wp-config.php file.
+Get the value of a specific variable or constant defined in wp-config.php
```

Comment:
Just first glance notice lack of third-person singular here and in list!

Reply:
Changed.